docs/IE_PLUGIN_DG/PluginTesting.md

```diff
@@ -41,18 +41,18 @@ To use these tests for your own plugin development, link the `IE::funcSharedTest
 To build test binaries together with other build artifacts, use the `make all` command. For details, see
 [Build Plugin Using CMake*](@ref openvino_docs_ie_plugin_dg_plugin_build).
 
-### Tests for plugin-specific ngraph transformations
+### Tests for plugin-specific OpenVINO™ transformations
 
-Please, refer to [Transformation testing](@ref ngraph_transformation) guide.
+Please, refer to [Transformation testing](@ref openvino_docs_transformations) guide.
 
 ### How to Extend Inference Engine Plugin Tests
 
 Inference Engine Plugin tests are open for contribution.
 Add common test case definitions applicable for all plugins to the `IE::funcSharedTests` target within the DLDT repository. Then, any other plugin supporting corresponding functionality can instantiate the new test.
 
-All Inference Engine per-layer tests check test layers functionality. They are developed using nGraph functions
+All Inference Engine per-layer tests check test layers functionality. They are developed using ov::Models
 as input graphs used by tests. In this case, to test a new layer with layer tests, extend
-the `IE::ngraphFunctions` library, which is also included in the Inference Engine Developer package, with a new nGraph function
+the `IE::ngraphFunctions` library, which is also included in the Inference Engine Developer package, with a new OpenVINO™ Model
```

Comment:
do we need `OpenVINO™` marketing name here?

Reply:
and please have a look in other places with ` OpenVINO™` in this PR. Sometimes we replaced `ngraph function` with ` OpenVINO™ model`, but can replace with just `model`